👋 Hello! Looks like there's a couple of unresolved components that result in build errors? ([link](https://app.circleci.com/pipelines/github/Automattic/wp-calypso/70875/workflows/8cdb5f33-f4d8-4cad-85fa-4b2682aa3d0d/jobs/856277)):

```
ModuleNotFoundError: Module not found: Error: Can't resolve '../wpcom/components/country-specific-payment-fields-ui' in '/home/circleci/wp-calypso/client/my-sites/checkout/composite-checkout/payment-methods'
    at /home/circleci/wp-calypso/node_modules/webpack/lib/Compilation.js:925:10
```

Also seeing:

```
 Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/circleci/wp-calypso/client/package.json (relative path: ./my-sites/checkout/composite-checkout/wpcom/components/country-specific-payment-fields-ui)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/circleci/wp-calypso/client/my-sites/checkout/composite-checkout/wpcom/components/country-specific-payment-fields-ui doesn't exist
      .json
```

--- a/client/my-sites/checkout/composite-checkout/payment-methods/netbanking.js
+++ b/client/my-sites/checkout/composite-checkout/payment-methods/netbanking.js
@@ -24,15 +24,15 @@ import { camelCase } from 'lodash';
  */
 import notices from 'notices';
 import { validatePaymentDetails } from 'lib/checkout/validation';
-import useCountryList from 'my-sites/checkout/composite-checkout/wpcom/hooks/use-country-list';
-import Field from 'my-sites/checkout/composite-checkout/wpcom/components/field';
+import useCountryList from 'my-sites/checkout/composite-checkout/hooks/use-country-list';
+import Field from 'my-sites/checkout/composite-checkout/components/field';
 import {
 	SummaryLine,
 	SummaryDetails,
-} from 'my-sites/checkout/composite-checkout/wpcom/components/summary-details';
-import { PaymentMethodLogos } from 'my-sites/checkout/composite-checkout/wpcom/components/payment-method-logos';
+} from 'my-sites/checkout/composite-checkout/components/summary-details';
+import { PaymentMethodLogos } from 'my-sites/checkout/composite-checkout/components/payment-method-logos';
 import { maskField } from 'lib/checkout';
-import CountrySpecificPaymentFieldsUI from '../wpcom/components/country-specific-payment-fields-ui';
+import CountrySpecificPaymentFieldsUI from '../components/country-specific-payment-fields-ui';
 
 const debug = debugFactory( 'composite-checkout:netbanking-payment-method' );
 

